test/e2e/app.spec.ts

```diff
@@ -1,6 +1,10 @@
 import {browser, by, element, ExpectedConditions} from 'protractor';
 
 describe('angular example application', () => {
+  beforeAll(function(done) {
+    setTimeout(done, 20000);
```

Comment:
there has to be a better answer.

we have
```
  getPageTimeout: 60 * 1000,
  allScriptsTimeout: 60 * 1000,
``` 
in `protractor.conf.js`, but I guess this is not enough. Do we need some help from @vikerman ?